courses/free-courses-es.md

```diff
@@ -84,6 +85,7 @@
 
 ### Control de Versiones
 
+* [Curso de GIT y GITHUB desde Cero para principiantes](https://www.youtube.com/watch?v=3GymExBkKjE&ab_channel=MoureDevbyBraisMoure) - Brias Moure (MoureDev)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Yes, but this video is over an hour long and is structured as an episodic course

Reply:
OK. When this is the case, use the PR description to let us know.

Reply:
Thank you for your comments. I have already added this information from the description